### PR TITLE
feat: Rating

### PR DESCRIPTION
**Pull request by SuyashTiwariCoder**
Added:
1) Star Ratings
2) Badge Ratings

### CODE REVIEW

**Comment by SuyashTiwariCoder** [comment]
rating1
rating 2

File: components/Ratings/rating.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    color: var(--success);
+}
+.fa-star:hover{
+    color: var(--danger);
+    cursor: pointer;
+}
+.rating--badge{
+    display: flex;
+
+
```
Comment (Kothalkarpriya) [inline components/Ratings/rating.css]: remove extra spaces

File: components/Ratings/rating.html [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+    </nav>
+    <div class="content">
+
+        <div class="sidebar">
+
+            <h3>Get Started</h3>
+            <div class="divider"></div>
+            <ul>
+                <li><a href="">Introduction</a></li>
+                <li><a href="">Colors</a></li>
```
Comment (Kothalkarpriya) [inline components/Ratings/rating.html]: use # if you don't have links to add right now

File: components/Ratings/rating.html [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+                <li><a href="">Image</a></li>
+                <li><a href="">xyz</a></li>
+            </ul>
+
+
+        </div>
+
+        <div class="main--content">
+
+
```
Comment (Kothalkarpriya) [inline components/Ratings/rating.html]: remove such extra spaces

File: components/Ratings/rating.html [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+</div>
+
+
+<div class="container">
+    <div class="code-wrapper">
+        <pre>
+
+    <code class="code">
+&lt;i class="fa fa-2x fa-star">&lt;/i>
```
Comment (Kothalkarpriya) [inline components/Ratings/rating.html:78]: Do you Formatting for code

**Review by Kothalkarpriya — CHANGES_REQUESTED** [review]
Please look into my comments

File: components/Ratings/rating.css [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+.fa-star:hover{
+    color: var(--danger);
+    cursor: pointer;
+}
+.rating--badge{
+    display: flex;
+}
+
+.badge--container{
+    background-color: white;
```
Comment (tusharkandpal) [inline components/Ratings/rating.css]: use variable here as well

File: components/Ratings/rating.html [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+<div class="showcase">
+
+    <i class="fa fa-2x fa-star"></i>
+    <i class="fa fa-2x fa-star"></i>
+    <i class="fa fa-2x fa-star"></i>
+    <i class="fa fa-2x fa-star"></i>
+    <i class="far fa-2x fa-star"></i>
+
+</div>
+
```
Comment (tusharkandpal) [inline components/Ratings/rating.html]: please format code!!

**Review by tusharkandpal — APPROVED** [review]

**Review by Kothalkarpriya — APPROVED** [review]
Good to go

**Comment by psychoactiv** [comment]
Looks good avoid using !important if possible as Vikram said in his last session.

**Pull request merged**